src/components/include/cos_kernel_api.h

```diff
@@ -107,5 +107,63 @@ int cos_hw_detach(hwcap_t hwc, hwid_t hwid);
 void *cos_hw_map(struct cos_compinfo *ci, hwcap_t hwc, paddr_t pa, unsigned int len);
 int cos_hw_cycles_per_usec(hwcap_t hwc);
 
+/* Default API */
+/* Capabilities for Async activation end point */
+struct cos_aep_info {
+	tcap_t    tc;
+	thdcap_t  thd;
+	arcvcap_t rcv;
+};
+
+/* Default Component information */
+struct cos_defci {
+	struct cos_compinfo ci;
+	struct cos_aep_info sched_aep;
+};
+
+/* current component's default component information */
+extern struct cos_defci curr_defci;
+
+/*
+ * cos_defci_init: initialize the current component's global cos_defci struct using the standard boot capabilities layout.
+ */
+void cos_defci_init(void);
```

Comment:
I'll be using `cos_get_heap_ptr()` that is used in the `micro_booter` and the `vkernel` `cos_compinfo_init()` for this default initialization.